tests/render/renderer/test_datasource_new_notebook_renderer.py

```diff
@@ -71,13 +71,15 @@ def _construct_datasource_new_notebook_assets(
             }
         ]
 
+        # this needs to be updated
         files_docs_cell = [
             {
                 "cell_type": "markdown",
                 "source": """### For files based Datasources:
-Here we are creating an example configuration using an InferredAssetDataConnector which will add a Data Asset for each file in the base directory you provided. This is just a sample, you may customize this as you wish!
+Here we are creating an example configuration.  The configuration contains an **InferredAssetFilesystemDataConnector** which will add a Data Asset for each file in the base directory you provided. It also contains a **RuntimeDataConnector** which can accept filepaths.   This is just an example, and you may customize this as you wish!
 
-See our docs for other methods to organize assets, handle multi-file assets, name assets based on parts of a filename, etc.""",
+Also, if you would like to learn more about the **DataConnectors** used in this configuration, including other methods to organize assets, handle multi-file assets, name assets based on parts of a filename, please see our docs on [InferredAssetDataConnectors](https://docs.greatexpectations.io/en/latest/guides/how_to_guides/configuring_datasources/how_to_configure_an_inferredassetdataconnector.html) and [RuntimeDataConnectors](https://docs.greatexpectations.io/en/latest/guides/how_to_guides/creating_batches/how_to_configure_a_runtime_data_connector.html). 
+""",
```

Comment:
This was a bit of clean up from the previous PR that modified the CLI notebooks.  There were 3 tests in this file in `test_datasource_new_notebook_renderer.py` that weren't passing

Reply:
these changes were included in #2845